ATGMediaBrowser/MediaBrowser/MediaBrowserViewController.swift

```diff
@@ -573,6 +602,24 @@ extension MediaBrowserViewController {
         controlViews.append(pageControl)
     }
 
+    private func addTitleLabel() {
+        view.addSubview(titleLabel)
```

Comment:
Better to add a newline after opening brace, to conform with coding style in rest of codebase.